src/robusta/patch/patch.py

```diff
@@ -0,0 +1,122 @@
+from typing import Union, List, Dict, get_type_hints
+from dataclasses import is_dataclass, InitVar
+from inspect import signature, getmodule
+from hikaru import HikaruDocumentBase, HikaruBase
+from ruamel.yaml import YAML
+
+try:
+    from typing import get_args, get_origin
+except ImportError:  # pragma: no cover
+    def get_args(tp):
+        return tp.__args__ if hasattr(tp, "__args__") else ()
+
+    def get_origin(tp):
+        return tp.__origin__ if hasattr(tp, "__origin__") else None
+
+NoneType = type(None)
+
+
+def create_monkey_patches():
+    print("Creating hikaru monkey patches")
+    HikaruBase.get_empty_instance = get_empty_instance
+    HikaruBase._get_hints = _get_hints
+    print("Creating yaml monkey patch")
+    YAML.official_plug_ins = official_plug_ins
+
+
+def official_plug_ins(self):
+    return []
+
+# hikaru meta.py monkey patch function
+@classmethod
+def get_empty_instance(cls):
```

Comment:
I would explicitly add a comment marking the areas that are your changes vs the orignal

Reply:
adding it